### ISSUE TITLE
【GDScript BUG】Enumeration variables have no code hints.

### ISSUE THREAD

**Issue by Atem1995** [godotengine/godot#71138]
### Godot version

4.0 Beta 10

### System information

Window10

### Issue description

![捕获](https://user-images.githubusercontent.com/52338846/211444968-aea2130e-35cb-4429-821c-c1c196f8ea44.PNG)

Enumeration variables have no code hints.

### Steps to reproduce

略

### Minimal reproduction project

略

**Comment by arkology** [comment]
Fixed by https://github.com/godotengine/godot/issues/61427?

**Comment by anvilfolk** [comment]
Yeah, this is a duplicate of #61427 and should be fixed by #69590 :)
